contrib/githooks/main-hook.sh

```diff
@@ -67,9 +67,9 @@ RUN_PYTEST=$([ "${SKIP_PYTEST}" = "1" ] && echo "0" || echo "1")
 # pytest
 [ "${RUN_PYTEST}" -eq "1" ] \
     && {
-        echo -e "\033[32mRunning \033[33mpytest_run.py \033[32mbefore ${_GIT_COMMAND}... \033[0m"
+        echo -e "\033[32mRunning \033[33mpytest \033[32mbefore ${_GIT_COMMAND}... \033[0m"
 
-        python pytest_run.py
+        python -m pytest sopel test/
```

Comment:
Is this truly equivalent to what was there before? Does running `pytest` this way include `@plugin.example()`s from core plugins?

Reply:
Yes, hence the `sopel` and `test/`. That's how I run tests locally.